src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/__tests__/components/summary/__snapshots__/SummaryGroupComponent.test.tsx.snap

```diff
@@ -0,0 +1,88 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/summary/MultipleChoiceSummary.tsx components/summary/MultipleChoiceSummary.tsx -- should match snapshot 1`] = `
+<DocumentFragment>
+  <div
+    class="MuiGrid-root MuiGrid-container"
+  >
+    <div
+      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10"
+    >
+      <span
+        class="MuiTypography-root makeStyles-label-1  MuiTypography-body1"
+      >
+        Mock group
+      </span>
+    </div>
+    <div
+      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
+    >
+      <button
+        class="MuiButtonBase-root MuiIconButton-root makeStyles-editButton-4"
+        tabindex="0"
+        type="button"
+      >
+        <span
+          class="MuiIconButton-label"
+        >
+          <p
+            class="MuiTypography-root makeStyles-change-6 MuiTypography-body1"
+          >
+            <span>
+              Change
+            </span>
+            <i
+              class="fa fa-editing-file makeStyles-editIcon-5"
+            />
+          </p>
+        </span>
+        <span
+          class="MuiTouchRipple-root"
+        />
+      </button>
+    </div>
+    <div
+      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
+    >
+      <div
+        style="padding-bottom: 24px;"
+      >
+        <p
+          class="MuiTypography-root MuiTypography-body1"
+        >
+          <span
+            class="makeStyles-label-7"
+          />
+        </p>
+        <p
+          style="margin-bottom: 0px; display: inline;"
+        >
+          Mock field 1
+        </p>
+         : 
+        <span>
+          1
+        </span>
+        <p />
```

Comment:
Er det en måte du kan ungå å ha tomme paragrafer i snapshoten på?

Reply:
Not sure, this seems like something the snapshot is doing, although I can't re-create it when rendering the component in an app 🤔 Can't see that there is any input to the component missing that should have been populated here either 🤷

Reply:
It's good that they are omitted on render. Though it might be just the browser doing it, (omitting empty containers).
Have you checked in other browsers?